### PR TITLE
FACES-2696 Utilize Portlet 3.0 dynamic resource dependency feature (backport to reduce diffs)

### PR DESCRIPTION
**Pull request by stiemannkj1**

### CODE REVIEW

**Comment by stiemannkj1** [comment]
@ngriffin7a,
Please backport to `3.x` **only**.

File: tck/bridge-tck-harness/src/main/resources/tomcat_pluto/pluto-portal-driver-config.xml [hunk 1]
```diff
@@ -1,652 +1,753 @@
...
+			<portlet context="/com.liferay.faces.demo.jsf.applicant.portlet" name="1"/>
+		</page>
+		<page name="jsf-cdi-applicant" uri="/WEB-INF/themes/pluto-default-theme.jsp">
+			<portlet context="/com.liferay.faces.demo.jsf.cdi.applicant.portlet" name="1"/>
+		</page>
+		<page name="jsf-events" uri="/WEB-INF/themes/pluto-default-theme.jsp">
+			<portlet context="/com.liferay.faces.demo.jsf.ipc.events.customers.portlet" name="customers"/>
+			<portlet context="/com.liferay.faces.demo.jsf.ipc.events.bookings.portlet" name="bookings"/>
+		</page>
+		<page name="jsf-flows" uri="/WEB-INF/themes/pluto-default-theme.jsp">
```
Comment (stiemannkj1) [inline tck/bridge-tck-harness/src/main/resources/tomcat_pluto/pluto-portal-driver-config.xml:66]: @ngriffin7a,
Please remove jsf-flows and html5 from this file in `3.x`.

**Comment by stiemannkj1** [comment]
Also consider deleting all the empty `Compat` classes. Not sure what our policy is on that, but I don't think that it would make backports harder.

**Comment by ngriffin7a** [comment]
Did not merge -- instead backported commits from master (6.x) branch.

**Pull request closed without merging**